wporg-gp-translation-events.php

```diff
@@ -461,7 +461,11 @@ public function generate_event_slug( array $data, array $postarr ): array {
 	 * @return void
 	 */
 	public function add_active_events_current_user(): void {
-		$user_attending_events      = get_user_meta( get_current_user_id(), Route::USER_META_KEY_ATTENDING, true ) ?: array();
+		$user_attending_events = get_user_meta( get_current_user_id(), Route::USER_META_KEY_ATTENDING, true ) ?: array();
```

Comment:
I agree with the solution in this PR being more explicit, so I would opt for it instead or setting `array( 0 )`.

Reply:
The `arrray( 0 )` solution is returning an array with and invalid ID (0), because if you pass an empty array to the query, it returns a post :/ 
Let's merge this PR.